frontend/src/app/actions/browser.js

```diff
@@ -0,0 +1,3 @@
+import { createActions } from 'redux-actions';
+
+export default createActions({}, 'setState');
```

Comment:
redux-actions is so polymorphic I have a hard time understanding what is going on. What is the return value here? Why is the first parameter empty? What is the point of this module if nothing ever imports it?